src/vis/vis.js

```diff
@@ -288,26 +285,26 @@ var VisModel = Backbone.Model.extend({
    * Force a map instantiation.
    * Only expected to be called once if {skipMapInstantiation} flag is set to true when vis is created.
    */
-  instantiateMap: function (options, includeFilters) {
+  instantiateMap: function (options) {
     options = options || {};
     if (!this._instantiateMapWasCalled) {
       this._instantiateMapWasCalled = true;
       var successCallback = options.success;
       options.success = function () {
-        this._dataviewsTracker.track();
+        this._dataviewsCollection.track();
```

Comment:
I kind of liked the dataview tracker that we had before: a class whose only responsibility is to trigger an event once all the given dataviews are fetched (I wasn't suggesting moving all the logic to the dataviewsCollection, but only `isAnyDataviewFiltered`...). Also, I'd rather trigger the `dataviewsFetched` from this class, so that it's not hidden somewhere...

```
var promise = DataviewStatus.waitUntilAllDataviewsFetched(this._dataviewsCollection.models);
promise.then(function () {
  this.trigger('dataviewsFetched');
}.bind(this));
```

Just an idea... We could do something similar with events too... what do you think?